images/win/scripts/Installers/Install-MongoDB.ps1

```diff
@@ -0,0 +1,9 @@
+####################################################################################
+##  File:  Install-MongoDB.ps1
+##  Desc:  Install MongoDB
+####################################################################################
+
+choco install mongodb
+
+Add-MachinePathItem "$($env:SystemDrive)\Program Files\MongoDB\Server\4.2\bin"
```

Comment:
Are you sure what 4.2 will not be changed?

Reply:
There's no way to be sure mongodb/choco team won't change the path. The validation script is a waring tool the path has to be changed